template/project.yml

```diff
@@ -1,6 +1,8 @@
 project:
   # This will be used as the vendor name in the new project's composer.json file.
   vendor_name: 'acquia'
+  # This will be used as the description in the new project's composer.json file.
+  description: 'A short description of the package. Usually this is just one line long.'
```

Comment:
Something like this `The project's composer.json will contain this value and should be updated directly to change this value after initial project creation`?